Source/WebCore/dom/EventTarget.h

```diff
@@ -169,6 +174,13 @@ class EventTarget : public ScriptWrappable, public CanMakeWeakPtr<EventTarget, W
             bitfield &= ~static_cast<uint16_t>(EventTargetFlag::HasEventTargetData);
         weakPtrFactory().setBitfield(bitfield);
     }
+
+    void setIsNode() const
```

Comment:
Can we set this flag in EventTarget's constructor?

Reply:
Sure